src/workarounds/roc_arrays.jl

```diff
@@ -0,0 +1 @@
+synchronize_device(::GPU{<:AMDGPU.ROCArray}) = AMDGPU.Device.sync_workgroup()
```

Comment:
`sync_workgroup` is meant to be called from a kernel.
AMDGPU has equivalent to CUDA `AMDGPU.synchronize` method.

```suggestion
synchronize_device(::GPU{<:AMDGPU.ROCArray}) = AMDGPU.synchronize()
```

Reply:
Thanks!